go/app/model/item.go

```diff
@@ -0,0 +1,64 @@
+package model
+
+import (
+	"database/sql"
+	_ "github.com/mattn/go-sqlite3"
+	"github.com/google/uuid"
+)
+
+type Item struct {
+	Name string `json:"name"`
+	Category string `json:"category"`
+}
+
+type Items struct {
+	Items []Item `json:"items"` 
+}
+
+
+
+func GetItems(db *sql.DB) ([]Item, error) {
+	var err error
+	cmd := "SELECT * FROM items"
+	rows, _ := db.Query(cmd)
+	defer rows.Close()
+	//structを作成
+	var item_list []Item
+	//取得したデータをループでスライスに追加　for rows.Next()
+	for rows.Next() {
+		var item Item
+		var id uuid.UUID
+		//scan データ追加
+		err = rows.Scan(&id, &item.Name, &item.Category)
```

Comment:
```suggestion
	cmd := "SELECT name, category FROM items"
	rows, _ := db.Query(cmd)
	defer rows.Close()
	//structを作成
	var item_list []Item
	//取得したデータをループでスライスに追加　for rows.Next()
	for rows.Next() {
		var item Item
		//scan データ追加
		err = rows.Scan(&item.Name, &item.Category)
```

`curl -X GET 'http://127.0.0.1:9000/items'`を実行すると、

以下のレスポンス、エラーログが出力されます。
```
{"message":"sql: Scan error on column index 0, name \"id\": Scan: unable to scan type int64 into UUID"}
```
```
{"time":"2022-05-19T15:12:10.212942+09:00","level":"ERROR","prefix":"echo","file":"main.go","line":"43","message":"sql: Scan error on column index 0, name \"id\": Scan: unable to scan type int64 into UUID"}
```
AddItem()同様、idをUUIDで受け取ろうとしているためです。
受け取ったidは使わないのであれば、クエリ文を変更してあげると良いかなと思いました！